test/test_docstring_detection.py

```diff
@@ -0,0 +1,44 @@
+import tokenize
+from unittest import TestCase
+
+from flake8_quotes import Token, get_docstring_tokens
+from test.test_checks import get_absolute_path
+
+
+class GetDocstringTokensTests(TestCase):
+    def _get_docstring_tokens(self, filename):
+        with open(get_absolute_path(filename), 'r') as f:
+            tokens = [Token(t) for t in tokenize.generate_tokens(f.readline)]
+        return get_docstring_tokens(tokens)
+
+    def test_get_docstring_tokens_absent(self):
```

Comment:
While these tests work, it feels weird to conflate the 2 sets of test files as it will cause extra work between the 2 test suites if we ever want to add a new edge case

We should have a different set of test files for verifying our docstring extraction works as expected; each with a single purpose (e.g. 1 file for classes with many different types of indentation, 1 file for modules (or I guess multiple))